_drafts/zero-to-main-libc.md

```diff
@@ -0,0 +1,721 @@
+---
+title: "From zero to main(): Bootstrapping libc with Newlib"
+description:
+  "TODO"
+author: francois
+tags: [zero-to-main]
+---
+
+In this series of post, we’ve worked methodically to demystify what happens to
```

Comment:
In the 'Zero to Main()' series of post[s]

Maybe add a link too.

Reply:
👍